fix(swap): handle quote response (#1125)
chainflip-io/chainflip-sdk-monorepo: packages/swap/python-client/README.md

#### Single leg

> Quoting server emits the following `quote_request` event to sell $FLIP for
> \$USDC:

Reviewer: ```suggestion
Quoting server emits the following `quote_request` event for a swap that 
sells $FLIP for $USDC:
```

}
```

> The response should contain orders for the opposite direction. For `SELL`
> orders, you should respond with `BUY` orders. The client receives the reponse
> and emits the following `quote_response` event:

Reviewer: ```suggestion
The response should contain orders for the opposite direction. For `SELL`
swap, you should respond with `BUY` orders. The client receives the response
and emits the following `quote_response` event:
```

Author: 🇫🇷

```jsonc
{
  "request_id": "018ee1c7-b949-71f2-889e-a2284cee7712",
  "legs": [
    [
      [-268100, "1137934"],
      [-268000, "1149370"],
    ],
  ],
}
```

> This response means you intend to open two `BUY` orders in response to the
> `SELL` request. These orders will buy approximately 0.5 $FLIP each, thus filling

Reviewer: ```suggestion
This response means you intend to open two `BUY` orders in response to the
`SELL` swap. These orders will buy approximately 0.5 $FLIP each, thus filling
```
